src/less/filter-menu/base/filter-menu.less

```diff
@@ -0,0 +1,99 @@
+span.filter-menu {
+    display: inline-block;
+    max-width: 288px;
+    min-width: 144px;
+}
+
+.filter-menu__items {
+    .dropdown-base;
+
+    border-color: @filter-menu-border-color;
+    border-radius: 16px;
+    overflow: hidden;
+}
+
+.filter-menu__scroll-area {
+    margin-top: 8px;
+    max-height: calc(50vh - 40px); // half of the viewport height minus the footer
+    overflow-y: auto;
+    position: relative;
+}
+
+span.filter-menu__items[role="menu"] {
+    display: inline-block;
+}
+
+div.filter-menu__item[role^="menuitem"] {
+    box-sizing: border-box;
+    cursor: default; // needed to override text cursor
+    display: inline-flex;
+    font-size: 0.875rem;
+    justify-content: space-between;
+    line-height: 1.4em;
+    padding: 8px 16px;
+    width: 100%;
+
+    &:focus {
+        outline: none;
+    }
+
+    &:focus,
+    &:hover {
+        background-color: @filter-menu-item-background-color;
+    }
+
+    &:active {
+        font-weight: @dropdown-item-active-font-weight;
+        // In future add a generic status class used by all dropdowns
+    }
+
+    &:last-child {
+        margin-bottom: 8px;
+    }
+
+    & > span:not(.checkbox) {
```

Comment:
Is this to identify the footer span? I think it would be more efficient just to use the footer class, unless there are other spans that need the same treatment.

Reply:
This is to overcome specificity issues, given we're using some "dropdown" mixin code.